docker/Caddyfile

```diff
@@ -0,0 +1,36 @@
+
+(log_common) {
+    log {
+        output file /var/log/caddy/{args.0}.access.log
+    }
+}
+
+# Change the host to your domain (this will serve at inventree.localhost)
+inventree.localhost {
```

Comment:
```suggestion
{$INVENTREE_SITE_URL:inventree.localhost} {
```

See for context https://caddyserver.com/docs/caddyfile-tutorial#environment-variables